src/components/Issue and Commits List/IssuesList.tsx

```diff
@@ -63,6 +86,19 @@ const IssuesList = ({ repo }: { repo: GithubRepo }) => {
           }}
         />
       ))}
+      <div className="footer text-center">
```

Comment:
What happens if you get an error? I assume you will still show these page controls.

We shouldn't ...